privacy.html

```diff
@@ -7,7 +7,7 @@ <h2 class="title">Privacy statement to members of the SRCF</h2>
 <h3 id="introduction">Introduction</h3>
 <p>This statement explains how the Student-Run Computing Facility (&quot;the SRCF&quot;, &quot;the Society&quot;, &quot;we&quot;, &quot;our&quot; or &quot;us&quot;) collects and handles personal data about you as a member of the Society, and our reasons for doing so.</p>
 <p>The latest version of this statement can be found on our website at <a href="<!--#if expr="${domain_web}" --><!--#echo var="domain_web" --><!--#endif -->/privacy">https://www.srcf.net/privacy</a>. The SRCF is the data controller for data held under this privacy statement. The SRCF Executive is the designated data protection contact, and are in charge of data protection matters. They can be contacted at <a href="mailto:data-protection@srcf.net">data-protection@srcf.net</a>.</p>
-<p><strong>Please note:</strong> this statement only applies to data held by the SRCF for which it is the data controller. The SRCF also hosts, on its systems, users and societies who may hold personal data; however it is not the data controller in these cases, as such users/societies act as their own data controllers. You should refer to the privacy statements of those users/societies where this applies to you as a data subject.</p>
+<p><strong>Please note:</strong> this statement only applies to data held by the SRCF for which it is the data controller. The SRCF also hosts, on its systems, users and groups who may hold personal data; however it is not the data controller in these cases, as such users/societies act as their own data controllers. You should refer to the privacy statements of those users/societies where this applies to you as a data subject.</p>
```

Comment:
Should we add to the "Edited at" lines at the top of this to mark the amendment (without material changes) made here?